src/integrationTest/resources/application.properties

```diff
@@ -15,3 +15,4 @@ scheduling.task.pending-notifications.enabled=false
 scheduling.task.notifications-consume.enabled=false
 
 idam.s2s-auth.url=false
+notifications.wait-duration-to-process-pending-in-minute = 60
```

Comment:
for int test can we have 0 ?

Reply:
for func test it should be 0 so in INT test we can test another situation.?